charts/nginx-gateway-fabric/values.yaml

```diff
@@ -72,15 +72,15 @@ nginx:
 
   ## The configuration for the data plane that is contained in the NginxProxy resource.
   config: {}
-    # disableHTTP2: false
```

Comment:
These should remain indented because they are nested under the `config` object.

Reply:
This is interesting...seems like `yamllint` doesn't like this use case of commenting only a part of the block...

Reply:
I'm assuming it's because we have the closed brackets, signifying that config is its own object, and then it assumes that the yaml underneath is separate. But it's technically part of the config block, we just indent to make it visible and easy to users.

Reply:
it doesn't complain if I put the `{}` on a new line aligned with the comments